kuksa_feeders/replay/stressClient.py

```diff
@@ -0,0 +1,55 @@
+import configparser
+import sys
+import argparse
+import time
+from pickle import FALSE
```

Comment:
why this import?

Reply:
stayed in there from previous versions, but now unnecessary. Will be deleted